templates/validators.html

```diff
@@ -116,7 +117,7 @@ <h6 class="dropdown-header"><span>Filter state</span> <span><span class="text-su
                 stateLoadCallback: function (settings) {
                   return JSON.parse(localStorage.getItem("DataTables_" + settings.sInstance))
                 },
-                ajax: '/validators/data?filterByState=' + state,
+                ajax: dataTableLoader('/validators/data', new URLSearchParams([["filterByState", state]])),
```

Comment:
Debouncing does not work if a filter is set

Reply:
This works in a weird way.
If you refresh the page and then change the filter it does not debounce further quick page changes.
But if you refresh the page with a filter already set the quick page changes ARE debounced.